exwm-workspace.el

```diff
@@ -252,26 +281,106 @@ The optional FORCE option is for internal use only."
       (xcb:+request exwm--connection
           (make-instance 'xcb:ChangeWindowAttributes
                          :window window-id :value-mask xcb:CW:EventMask
-                         :event-mask xcb:EventMask:SubstructureRedirect))))
-  (xcb:flush exwm--connection)
-  ;; We have to delay making the frame visible until the
-  ;; override-redirect flag has been set.
-  (select-frame-set-input-focus (car exwm-workspace--list))
-  (dolist (i exwm-workspace--list)
-    (set-frame-parameter i 'visibility t)
-    (lower-frame i)
-    (set-frame-parameter i 'fullscreen 'fullboth))
-  (raise-frame (car exwm-workspace--list))
+                         :event-mask xcb:EventMask:SubstructureRedirect)))
+    (xcb:flush exwm--connection)
+    ;; We have to delay making the frame visible until the
+    ;; override-redirect flag has been set.
+    (set-frame-parameter frame 'visibility t)
+    (lower-frame frame)
```

Comment:
To be honest... I don't know.  I have tested commenting the line and see no difference neither w.r.t. the initial workspaces nor when adding/removing... Shall I drop it?